app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -33,6 +33,7 @@ export LC_ALL=C LANG=C
 <%= "\n# Repository data: [#{@repo_data}]" if @repo_data.present? -%>
 <%= "\n# Force: [#{@force}]" unless @force.nil? -%>
 <%= "\n# Ignore subman errors: [#{@ignore_subman_errors}]" unless @ignore_subman_errors.nil? -%>
+<%= "\n# Set up container certs: [#{@setup_container_registry_certs}]" unless @setup_container_registry_certs.nil? -%>
```

Comment:
Should this be hidden on non-katello?
```suggestion
<%= "\n# Set up container certs: [#{@setup_container_registry_certs}]" if plugin_present?('katello') && !@setup_container_registry_certs.nil? -%>
```

Reply:
This field is injected by katello so it will always be nil for non-katello installs like the other katello fields in the template.